docs/_providers/bind.md

```diff
@@ -5,23 +5,20 @@ layout: default
 jsId: BIND
 ---
 # BIND Provider
-
-This provider simply maintains a directory with a collection of .zone files. We currently copy zone files to our production servers and restart bind via
-a script external to DNSControl.
+This provider maintains a directory with a collection of .zone files. We currently copy zone files to our production servers and restart bind via a script external to DNSControl.
```

Comment:
Please change "We currently..." to "Pushing the zone files is left to you as different sites do this differently. Commonly people run a bash script that copes the files to the master and runs the proper command to notify the server."

Reply:
I'll fix this and finish with the rest in a few hours.

Reply:
Fixed